src/components/banner/style.scss

```
@@ -0,0 +1,20 @@
+.wpui-lubus{
+    background-color: rgba(78, 78, 78, 0.38);
+    padding: 7px;
+    padding-left: 12px;
+    padding-right: 12px;
```

```suggestion
    padding: 7px 12px;
```

we can use the shorthand